app/dashboard/subDashboards/browser-based/ui.test.js

```diff
@@ -75,7 +75,7 @@ test('should render all the sections for the Browser based sections section grou
     .expect(mobileFirstSection.find('[data-test-id="dashboard-section-title"]').innerText)
     .eql('Mobile first')
     .expect(mobileFirstSection.find('[data-test-id="dashboard-section-title"] a').getAttribute('href'))
-    .eql('/solution/S100000-001/section/browser-mobile-first')
+    .eql('/solution/S100000-001/dashboard/browser-based/section/browser-mobile-first')
```

Comment:
do we still need /section/?

Reply:
yes we do. We still need to point to the section we want to go to. This just tells the route that it's come from a subDashboard.
The solution we went for is to structure the url (like we did with public browse) to reflect the nested dashboard rather than the flat structure we had before.

Reply:
👍